docs/GettingStarted.md

```diff
@@ -566,6 +595,12 @@ Datadog::Tracing.trace('outer') do
 end
 ```
 
+`options` are the following keyword arguments:
+
+| Key       | Env Var                         | Type   | Description                                  | Default |
+| --------- | ------------------------------- | ------ | -------------------------------------------- | ------- |
+| `enabled` | `DD_TRACE_CONCURRENT_RUBY_ENABLED` | `Bool` | Whether the integration should create spans. | `true` |
```

Comment:
The description is not accurate: `concurrent-ruby` integration does not generate spans.

Reply:
Fixed, thank you!